evm_loader/lib/src/rpc/db_call_client.rs

```diff
@@ -88,6 +89,6 @@ impl Rpc for CallDbClient {
     }
 
     async fn get_deactivated_solana_features(&self) -> ClientResult<Vec<Pubkey>> {
-        Ok(vec![]) // TODO
+        get_deactivated_features(self, Some(self.slot)).await
```

Comment:
You have to use `CloneRpcClient` to get_deactivated_features or the logic of it will be broken.